### ISSUE TITLE
Please revise japanese clip's ReadMe and the position of model.eval in cli.py

### ISSUE THREAD

**Issue by jackee777** [LAION-AI/CLIP_benchmark#84]
I wonder if one intorduction and one code may be something wrong.

At first,  japanese clip's ReadMe is not correct.
The exiting example is following.
```
python3 clip_benchmark/cli.py \
  --model_type "ja_clip" \ # flag to use japanese-clip
  --pretrained "rinna/japanese-cloob-vit-b-16" \ # now, we have `rinna/japanese-cloob-vit-b-16` or `rinna/japanese-clip-vit-b-16`. 
  --language "jp" \
  --task "zeroshot_classification"  \
  --dataset "imagenet1k" 
  --dataset_root {ROOT_PATH} 
```
However, it lacks eval/build and "\\".
Maybe, the following is correct. In addition to this, "#" or unnecessary spaces prevent executing this shell script.
Please check it. 
```
python3 clip_benchmark/cli.py eval \
  --model_type "ja_clip" \
  --pretrained "rinna/japanese-cloob-vit-b-16" \
  --language "jp" \
  --task "zeroshot_classification"  \
  --dataset "imagenet1k" \
  --dataset_root {ROOT_PATH} 
```

Secondly, [model.eval() ](https://github.com/LAION-AI/CLIP_benchmark/blob/c4d99276edfcc3c7559a92323cd3f7a374b22c06/clip_benchmark/cli.py#L181) in line. 181 of cli.py seems like unnecesary.
model.eval() has alerady been executed in load_clip() function.

I hope you to review them.

**Comment by mehdidc** [comment]
Thanks @jackee777! you are right, will update the example. On the other hand not sure to see where `model.eval()` is called in `load_clip`, if `model.training` is evaluated after clip, it is still `True`, so `mode.eval()` would still be necessary (for models using `dropout` and/or` batchnorm`).

**Comment by jackee777** [comment]
@mehdidc 
Thank you for your response. You are correct, and I'm sorry for misunderstanding the 2nd question.
I may have seen `load_clip` in another version of this repository. The current version works successfully.

I close this comment.
